### ISSUE TITLE
Dcmtk data dictionary file not preserved into docker image

### ISSUE THREAD

**Issue by ryansanford** [scitran/reaper#58]
This dictionary is required for the use of tag names, rather than grou/element codes

**Error**
```
root@04aee6315aee:/# dicom_sniper -y --secret secret -k StudyID "" www.dicomserver.co.uk 104 3333 REAPER DCMQRSCP https://localhost:8443 --loglevel DEBUG
findscu -v -S -aet REAPER -aec DCMQRSCP -k QueryRetrieveLevel="STUDY" -k StudyDate="" -k PatientID="" -k NumberOfSeriesRelatedInstances="" -k StudyID="" -k StudyInstanceUID="" -k SeriesInstanceUID="" -k StudyTime="" www.dicomserver.co.uk 104
findscu -v -S -aet REAPER -aec DCMQRSCP -k QueryRetrieveLevel="STUDY" -k StudyDate="" -k PatientID="" -k NumberOfSeriesRelatedInstances="" -k StudyID="" -k StudyInstanceUID="" -k SeriesInstanceUID="" -k StudyTime="" www.dicomserver.co.uk 104
W: no data dictionary loaded, check environment variable: DCMDICTPATH
I: Requesting Association
I: Association Accepted (Max Send PDV: 65524)
E: Bad override key/path: QueryRetrieveLevel=STUDY: Unable to parse tag/dictionary name at beginning of path: QueryRetrieveLevel
E: Find SCU Failed: 0001:0019 Unable to parse tag/dictionary name at beginning of path: QueryRetrieveLevel
I: Aborting Association

```

**Comment by ryansanford** [comment]
@gsfr Accidentially pushed fix for this (365e5492f77b7f294ce06bb80d75b09278c87b84) to master. Can you help undo that for proper PR, or just leave it?

**Comment by gsfr** [comment]
@ryansanford Just leave it.

Closed via 365e5492f77b7f294ce06bb80d75b09278c87b84.
